projects/@shared/components/properties-item.vue

```diff
@@ -198,10 +195,7 @@ export default {
       return Array.isArray(this.term);
     },
     warningMessage() {
-      if (this.warnings && this.warnings.length > 0) {
-        return this.warnings[0].message;
-      }
-      return null;
+      return this.warnings?.[0]?.message ?? null;
```

Comment:
I think the `?? null` is obsolete when using optional chaining, because as soon as any part doesn't exist it returns `undefined` which works just as well as `null` in our case.